### ISSUE TITLE
Create marketing package for partners

### ISSUE THREAD

**Issue by holanita** [Green-Software-Foundation/state-of-green-software#43]
What: we want to create a marketing package for stakeholders like CNCF to help them promote SOGS
Why: to boost the impact of big organizations like CNCF
When: package and design assets to be finalized by Nov 14 (comms will start on Nov 18)

Next steps:

- [x] Namrata to list what needs to be included in that package
- [x] N/A Anita/Namrata to work with Jenya on eventual design assets
- [x] Missing: Namrata to create the actual package asap
- [ ] Delayed Nov 28 (original: Nov 14) Asim approved the member package, Anita to check the adjusted partner package
- [ ] Delayed Nov 28 (original: Nov 18) Asim to email the package to these partners

Partners:
- CNCF, Hyperledger, FinOps, .Net Foundation, Python Software Foundation
- Microsoft (Green Community), Amazon, IBM, Salesforce, Google

**Comment by holanita** [comment]
@NAMRATA-WOKE please help figure out the content of this package so we can send it out in due time. As discussed with Asim yesterday, the package and design assets should be finalized by Nov 14 and comms will start on Nov 18.

**Comment by holanita** [comment]
@NAMRATA-WOKE can you list what you need for this package? What should be included? We were going to finish this by Nov 14 today, so it would be good to get started asap. Let's discuss in our call today if needed.

**Comment by holanita** [comment]
List of partners suggested by Asim and added to the initial comment above.

**Comment by holanita** [comment]
@NAMRATA-WOKE you were going to create a package for partners similar to the one mentioned above. Do you have that new file somewhere already? As for the file you already have, it would be nice to keep it as is, as members might ask for guidance later so it's good to have something to share with them. 

Next steps:
- [ ] Namrata to create package for partners
- [ ] Asim to review package and confirm final list of partners and how/who will reach out to them
- [ ] Anita to create shortlink for partners and set up tracking before the package is sent out

**Comment by holanita** [comment]
@jawache @NAMRATA-WOKE okay noted. just wanted to highlight that this ticket is about partners like CNFC. these two things have been mixed up since the beginning so just wanted to make sure we send the right stuff to the right audience.

**Comment by NAMRATA-WOKE** [comment]
@holanita i can finalize this package today I just need to know if they are also getting custom URLs either for internal or external use.

The last time we spoke, my recommendation was to provide them custom collector links for external outreach, versus internal since their organizations might be small but their networks massive!

**Comment by holanita** [comment]
@NAMRATA-WOKE unsure which package you mean by 'this' package as the ticket is about _partners_, but Asim was talking about _members_ above

- **package for partners** (by _partners_ we initially meant orgs like CNFC) - do you have this package approved by Asim already? let me know if you do and I will create a survey monkey collector and shortio link for you.
- **package for members** (_members_ are our own members) - we have the external link already. once (if) the package is approved by Asim, then it's good to go. I'm not quite sure whether it ism since I was away for some days, but it's his call for sure.

**Comment by NAMRATA-WOKE** [comment]
@holanita I was talking about partners. Yes it's approved I need the appropriate shortlinks to finalize the doc. All sorted on the members front.

**Comment by holanita** [comment]
@NAMRATA-WOKE so you need two links? one for external and one for internal? or two for each partner? and who are they in the end? please let me know what exactly has been agreed with Asim. I don't have access to the partner package so I am not aware of the details. it's really difficult to work without knowing the context

**Comment by NAMRATA-WOKE** [comment]
@holanita you were the one who showed me the list that Asim added to- I think it was a word doc. I'll try and find it tomorrow. Hard to navigate on my phone and I'll be away from my computer for the rest of the night. 

My recommendation is to measure the # of responses we get from each non-member Asim reaches out to personally. I think one collector for internal/external is fine. I wouldn't offer them the same reports as you are to members.

@jawache are you okay with the above?

**Comment by holanita** [comment]
@NAMRATA-WOKE let me have a quick call with @jawache because this is becoming way too messy. if Asim hasn't confirmed the list in my absence, then this is not good to go yet. I thought you guys discussed this recently.

**Comment by holanita** [comment]
@NAMRATA-WOKE all good, here is a quick summary on what we discussed with Asim just now:
- Asim will send out the emails to the partners listed above in my first comment. 
- **CNCF** and **Hyperledger** will get custom urls as they are the most likely to generate traffic.
- For the others, we just need one url (no need to separate internal and external channels!)

Could you duplicate your member package and adjust the wording a bit based on the above and to make it work for partners? Let me know when you have the new google doc and I'll take a quick look and will add the new urls too.

(Members are a bit different, so I've opened [a new ticket](https://github.com/Green-Software-Foundation/state-of-green-software/issues/66) to keep things separate and avoid confusion.)

**Comment by NAMRATA-WOKE** [comment]
Sounds good. I can have that ready in a couple of hours so will share the doc in the new ticket you created.

**Comment by holanita** [comment]
@NAMRATA-WOKE the sponsorship idea that we talked about yesterday has been cancelled in the end. I've updated the file and [added you to some more companies](https://docs.google.com/spreadsheets/d/1x_R9Z0qIn7WGvSA_tuW2QHiJIOBjlR0OhjzuLg6aFFk/edit?disco=AAAAken_9Xo). Can you reach out to all of them and mark them when you are done? I'll take care of the ones assigned to me. Thanks!

**Comment by NAMRATA-WOKE** [comment]
Partner Packages are good to go. I have started sending them out. In addition to Green Web Foundation, I reached out to Software Sustainability Institute and Ismael to repost on behalf of Adora Foundation. I am doing research to see which other organizations I can email to notify them of the survey.

**Comment by holanita** [comment]
Ah I see. Awesome. Did you get the emails/POCs from Asim? Let me know if you hear from any of them.

**Comment by holanita** [comment]
@NAMRATA-WOKE this ticket has become a bit too long and difficult to track. Let me create a new ticket for tracking our activities and conversions.

**Comment by NAMRATA-WOKE** [comment]
In addition to Green Web Foundation, I reached out to Software Sustainability Institute, DEV Community and BFD community.

Can also reach out to Founder world to see if they can send a note about the survey to their Developer Network meetup (7,582 members): https://www.meetup.com/developer-network/
